benchmark/multithread.cpp

```diff
@@ -65,15 +65,15 @@ static void mt_alloc_free(poolCreateExtParams params,
     auto values = umf_bench::measure<std::chrono::milliseconds>(
         bench.n_repeats, bench.n_threads,
         [&, pool = pool.get()](auto thread_id) {
-            for (int i = 0; i < bench.n_iterations; i++) {
+            for (int i = 0; i < (int)bench.n_iterations; i++) {
```

Comment:
please change the "i" type instead casting n_iterations

Reply:
Done